src/App.tsx

```diff
@@ -2,43 +2,81 @@ import { ChangeEvent, useState } from 'react';
 import styles from './App.module.scss';
 import TasksList from './components/TasksList';
 import NewTask from './components/NewTask';
-
-
+import 'semantic-ui-css/semantic.min.css';
 
 function App() {
-    const [tasksArray, setTasksArray] = useState([{ id: 0, item: "Buy Milk" }, { id: 1, item: "Go Run" }, { id: 2, item: "Make Dinner" }, { id: 3, item: "Bake Cake" }]);
-    const [newTaskItem, setNewTaskItem] = useState('')
+    const [tasksArray, setTasksArray] = useState([
+        { id: 0, item: 'Buy Milk' },
+        { id: 1, item: 'Go Run' },
+        { id: 2, item: 'Make Dinner' },
+        { id: 3, item: 'Bake Cake' },
+    ]);
+    const [newTaskItem, setNewTaskItem] = useState('');
+    const [editedTaskItem, setEditedTaskItem] = useState('');
```

Comment:
editedTaskItem sounds like a function. What does it hold and what is its purpose?

Reply:
it holds the new value of edited task. Meaning, this task was in the list already and I edited its content.